test/core/security/ssl_server_fuzzer.cc

```diff
@@ -97,9 +102,11 @@ extern "C" int LLVMFuzzerTestOneInput(const uint8_t* data, size_t size) {
     auto handshake_mgr =
         grpc_core::MakeRefCounted<grpc_core::HandshakeManager>();
     sc->add_handshakers(grpc_core::ChannelArgs(), nullptr, handshake_mgr.get());
```

Comment:
Suggest moving this line right before the call to `DoHandshake()` and using the same channel args for both this call and that one, since that's more consistent with how these APIs will be used in the production code.